includes/definitions.inc.php

```diff
@@ -766,7 +766,11 @@
 $config['os'][$os]['icon']             = 'ceragon';
 $config['os'][$os]['over'][0]['graph'] = 'device_bits';
 $config['os'][$os]['over'][0]['text']  = 'Traffic';
-
+$config['os'][$os]['over'][1]['graph'] = 'device_ceraos_RxLevel';
+$config['os'][$os]['over'][1]['text']  = 'Rx Level';
+$config['os'][$os]['over'][2]['graph'] = 'device_ping_perf';
+$config['os'][$os]['over'][2]['text']  = 'Ping Times';
+$config['os'][$os]['mib-dir'][]        = 'ceraos';
```

Comment:
It would be mib_dir but this isn't needed anyway as a directory of the same os name is checked anyway so you can just remove this line.